### DISCUSSION TITLE
Support for PostgreSQL `ltree`

### DISCUSSION THREAD

**Discussion by LeoBorai** [SeaQL/sea-orm#1476] [Q&A]
Hi! I have been using SeaQL with PostgreSQL and wanted to know if its possible to use `ltree` extension from PostgreSQL.

Reading the docs I didn't found any references to this type.

Reading through discussions here I found this entry:

https://github.com/SeaQL/sea-orm/discussions/652

Where a user did good progress on it but for some reason it wasn't merged.

If SeaQL doesn't plan to have support for LTree, is there any way I could reuse the LTree implementation from SQLx?

https://docs.rs/sqlx/latest/sqlx/postgres/types/struct.PgLTree.html

Thanks in advance!

**Comment by billy1624** [comment]
A status updates for newcomers
- https://github.com/SeaQL/sea-query/pull/604

**Reply by holmofy** [reply]
@billy1624 How to use in sea-orm? 


Can it be used like this?
```rust
#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Serialize, Deserialize)]
#[sea_orm(table_name = "paper_question")]
pub struct Model {
    #[sea_orm(primary_key, auto_increment = false)]
    pub paper_id: i32,
    #[sea_orm(primary_key, auto_increment = false)]
    pub question_id: i32,
    pub sort: i16,
    #[sea_orm(column_type = "LTree")]
    pub keypoint_path: String,
    #[sea_orm(column_type = "Float")]
    pub correct_ratio: f32,
}
```
